sound/soc/sof/topology.c

```diff
@@ -2571,9 +2572,7 @@ static int sof_link_dmic_load(struct snd_soc_component *scomp, int index,
  */
 static int sof_link_hda_process(struct snd_sof_dev *sdev,
 				struct snd_soc_dai_link *link,
-				struct sof_ipc_dai_config *config,
-				int tx_slot,
-				int rx_slot)
+				struct sof_ipc_dai_config *config)
```

Comment:
Is it possible to remove those link_hda related here in topology if it is processed in BE hw_params and hda_dai_config IPC sent there?